pkg/scheduler/ugm/manager.go

```diff
@@ -231,6 +240,15 @@ func (m *Manager) GetGroupsResources() []*GroupTracker {
 	return groupTrackers
 }
 
+func (m *Manager) GetGroupResourcesByGroupName(group string) *GroupTracker {
```

Comment:
Same like earlier

Reply:
Also already have a fuction with this name:
```
func (m *Manager) GetGroupResources(group string) *resources.Resource {
	m.lock.RLock()
	defer m.lock.RUnlock()
	if m.groupTrackers[group] != nil {
		return m.groupTrackers[group].queueTracker.resourceUsage
	}
	return nil
}
```

Reply:
Sorry :). Purpose of the above method is to get the user/group tracker object. So, GetUserTracker and GetGroupTracker fits in?

Reply:
Thank you for quick response! Good suggestion about GetUserTracker and GetGroupTracker!